src/comments.js

```diff
@@ -381,6 +383,20 @@ function handleConditionalExpressionComments(enclosingNode, followingNode, comme
   return false;
 }
 
+function handleAssignmentPatternComments(enclosingNode, comment) {
```

Comment:
I don't think that this is the right fix. Can you figure out where the comment is attached without this logic and find out why this node didn't print its comment?